de.dashup.application/src/main/webapp/widgets/dashup-share-banner.js

```diff
@@ -0,0 +1,45 @@
+import {LitElement, html, css} from 'https://unpkg.com/lit-element@2.1.0/lit-element.js?module'
+export class DashupShareBanner extends LitElement{
```

Comment:
Is this a widget that a user can add to his dashup? If so, why is this a web component inheriting `LitElement`? I thought we have agreed that our widgets are only plain HTML which contains our dashup components exclusively.

Reply:
Indeed, this is a widget. You could also leave the surrounding class away, but then you would need to write the html code in the render function inside of your document. This way you would only need to write <dashup-share-banner>. It's just more encapsulated.

Reply:
See the conversation in issue #85.